PCGen-Formula/code/src/java/pcgen/base/formula/parse/Operator.java

```diff
@@ -52,9 +52,9 @@ public String getSymbol()
 		return symbol;
 	}
 
-	private String symbol;
+	private final String symbol;
 
-	private Operator(String op)
+	Operator(String op)
```

Comment:
This isn't that important to me. I've seen "public" on an enum which is terrible.